rest/data/host.go

```diff
@@ -40,6 +41,20 @@ func (hc *DBHostConnector) FindHostsById(id, status, user string, limit int) ([]
 	return hostRes, nil
 }
 
+func (hc *DBHostConnector) FindHostsInRange(createdBefore, createdAfter time.Time, user, distro, status string, userSpawned bool) ([]host.Host, error) {
+	hostRes, err := host.FindHostsInRange(createdBefore, createdAfter, user, distro, status, userSpawned)
+	if err != nil {
+		return nil, err
+	}
+	if len(hostRes) == 0 {
+		return nil, gimlet.ErrorResponse{
+			StatusCode: http.StatusNotFound,
```

Comment:
Is a 404 really the right error code here? I think of HTTP status codes as being about the _request_, not its content. This is consistent with Mozilla's description https://developer.mozilla.org/en-US/docs/Web/HTTP/Status. If there were 0 hosts, I think that a 200 makes more sense semantically, since the request was completed successfully, even though it was a noop.

Reply:
now it will return a 200